hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/om/TestOzoneManagerHA.java

```diff
@@ -779,10 +780,12 @@ public void testReadRequest() throws Exception {
       OzoneManager ozoneManager = cluster.getOzoneManager(i);
       String omHostName = ozoneManager.getOmRpcServerAddr().getHostName();
       int rpcPort = ozoneManager.getOmRpcServerAddr().getPort();
+      // Test can pass without this line below
+      conf.set(OZONE_OM_ADDRESS_KEY, omHostName + ":" + rpcPort);
```

Comment:
Since this line is redundant now, can we remove it?

Reply:
Also, I think it would be a good idea to check here that after _omFailoverProxyProvider.performFailoverIfRequired(ozoneManager.getOMNodeId());_, the current OM proxy node in FailoverProvider is the expected nodeId.

Reply:
Removed the `conf.set` in commit https://github.com/apache/hadoop-ozone/pull/148/commits/3d14652045adbe0a7c40f2f6ed6a76995ef8220d.

Reply:
It seems the leader node id is already checked by the end of the for loop? Do you suggest adding another check?